adafruit_ble/characteristics/__init__.py

```diff
@@ -10,6 +10,8 @@
 
 from __future__ import annotations
 
+# pylint: disable=too-many-arguments
```

Comment:
Is there a central place this can be disabled?

Reply:
In another I PR, I changed `max-args=` in `.pylintrc`, which makes more sense for this kind of override.